source/administration/batch-framework.rst

```diff
@@ -0,0 +1,153 @@
+
+Overview
+--------
+
+The MinIO Batch Framework allows you to create, manage, monitor, and execute jobs using a YAML-formatted job definition file.
+The batch jobs run directly on the MinIO deployment to take advantage of the deployment's server-side processing power without constraints of the local machine where you run the :ref:`MinIO Client <minio-client>`.
+
+A batch file can define any number of job tasks.
+MinIO does not limit on the number of job tasks that you can define in a batch file.
+
+Once started, MinIO begins processing the job immediately, depending on resources available to the deployment.
+The job runs to completion.
+If any portion of the job fails, MinIO retries the job up to the number of times defined in the job definition.
+
+A batch file only runs to completion one time.
+You can :mc-cmd:`~mc batch start` the batch file as many times as desired to complete it again.
+
+The MinIO Batch Framework supports the following job types:
+
+.. list-table:: 
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%   
+
+   * - Job Type
+     - Description
+
+   * - ``replicate``
+     - Perform a one-time replication procedure from one MinIO location to another MinIO location.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+- Install the :ref:`MinIO Client <minio-client>`
+- Define an :mc-cmd:`alias <mc alias set>` for the MinIO deployment
+
+The :mc:`mc batch` commands include
+
+.. list-table::
+   :widths: 30 70
+   :width: 90%
+
+   * - :mc:`mc batch generate`
+     - .. include:: /reference/minio-mc/mc-batch-generate.rst
+          :start-after: start-mc-batch-generate-desc
+          :end-before: end-mc-batch-generate-desc
+   * - :mc:`mc batch start`
+     - .. include:: /reference/minio-mc/mc-batch-start.rst
+          :start-after: start-mc-batch-start-desc
+          :end-before: end-mc-batch-start-desc
+   * - :mc:`mc batch list`
+     - .. include:: /reference/minio-mc/mc-batch-list.rst
+          :start-after: start-mc-batch-list-desc
+          :end-before: end-mc-batch-list-desc
+   * - :mc:`mc batch status`
+     - .. include:: /reference/minio-mc/mc-batch-status.rst
+          :start-after: start-mc-batch-status-desc
+          :end-before: end-mc-batch-status-desc
+   * - :mc:`mc batch describe`
+     - .. include:: /reference/minio-mc/mc-batch-describe.rst
+          :start-after: start-mc-batch-describe-desc
+          :end-before: end-mc-batch-describe-desc
+
+Job Types
+---------
+
+``replicate``
+~~~~~~~~~~~~~
+
+The ``replicate`` job type performs a single-run replication of objects from one MinIO deployment or location to another MinIO deployment or location.
+The definition file can limit the replication by bucket, prefix, and/or filters to only replicate certain objects.
+
+Key advantages of doing replication with a batch job include:
+
+- Removes the client to cluster network as a potential bottleneck
+- No special permissions for the user launching the job, as it runs entirely server side on the cluster
```

Comment:
:thinking: do we not require any permissions for the Alias with which the command runs?

Reply:
Clarified with Harsha.
Added three new admin policies to the policy doc, added a security section to the framework doc, and reworked this line.